source/coding-the-legislation/writing_yaml_tests.md

```diff
@@ -146,13 +146,13 @@ Values can be arithmetic expressions too.
 
 ## Running a test
 
-To run YAML tests, use the command line tool `openfisca test`, documented [here](https://openfisca.readthedocs.io/en/latest/openfisca-run-test.html):
+To run YAML tests, use the command line tool `openfisca test`, documented [here](../openfisca-python-api/openfisca-run-test.html):
 
 ```sh
 openfisca test path/to/file.yaml
 ```
 
->You can also run tests programatically using the [`test_runner` module](https://openfisca.readthedocs.io/en/latest/test_runner.html).
+>You can also run tests programatically using the [`test_runner` module](../openfisca-python-api/test_runner.html).
```

Comment:
The syntax of those two updated links (`here` and `test runner module`) seems identical but running `make html` locally gives the following result in generated HTML:
* `here`: openfisca-python-api/openfisca-run-test.html
* `test runner module`: ../openfisca-python-api/test_runner.html

Reply:
Aha! The difference seems to be caused by the use of backticks inside the second link to format the word "test_runner" as code, and indeed recommonmark seems to have been [struggling with this for a while](https://github.com/commonmark/CommonMark/issues/439).

Version 0.5 of recommonmark behaves more as we want (generating relative links), so it might have been worth investigating using it and fixing [what it breaks](https://github.com/openfisca/openfisca-doc/pull/184#pullrequestreview-195748018), *but* it silently drops the code formatting.

At this point I'm despairing a bit.

Reply:
@sandcha suggested moving the code formatting out of the link title (by also moving the word "module" outside the link altogether) and that works.

I'm investigating what is broken by recommonmark 0.5, it seems that it doesn't handle well links with the `.md` extension in the source `.md` when these links have an anchor; in that case we have to explicitly use `.html` instead. Despair receding, hope dawning.

Reply:
Despairing again - when you replace `.md` extensions with `.html`, Sphinx emits warnings that it can"t check the links.

Reply:
I don't think this link brings much value. Programmatically running tests seems too advanced for this section, and I don't think no one has even done it.